plugins/kubernetes-backend/src/service/KubernetesFanOutHandler.ts

```diff
@@ -147,7 +147,11 @@ export interface KubernetesFanOutHandlerOptions
 export interface KubernetesRequestBody extends ObjectsByEntityRequest {}
 
 const isPodFetchResponse = (fr: FetchResponse): fr is PodFetchResponse =>
-  fr.type === 'pods';
+  fr.type === 'pods' ||
+  (fr.type === 'customresources' &&
+    fr.resources.length > 0 &&
+    fr.resources[0].apiVersion === 'v1' &&
+    fr.resources[0].kind === 'Pod');
```

Comment:
Should this look for any occurance of pod rather than the first element in the resources array?

Reply:
How could there be different kinds of resources in a `customresources` group?

Reply:
The resources in the `customresources` could be any resource, it could be:

- 0: ingress 
- 1: Pod

or 

-0: Pod
-1: Ingress 

in which case this would not work

Reply:
AFAIK, there could be multiple `customresources` groups, but each of them could be only one kind:

```jsonc
[
  {
    "type": "customresources",
    "resources": [] // Ingress
  },
  {
    "type": "customresources",
    "resources": [] // Pod
  }
]
```

Reply:
ping @mclarke47

Reply:
> AFAIK, there could be multiple customresources groups, but each of them could be only one kind:

I'm not so sure that's correct

Reply:
@mclarke47 Or can you provide any false positive?

https://github.com/backstage/backstage/blob/22c4ecc6c75eacd5d6de94dfa5feff988def3018/plugins/kubernetes-backend/src/service/KubernetesBuilder.ts#L245-L255

Reply:
@mclarke47 News?

Reply:
Sorry for the delay, I'll look into this, this week

Reply:
I think you're right, I must have been thinking of the request instead of the response here